packages/code-du-travail-api/src/server/routes/__tests__/suggest.spec.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const Koa = require("koa");
+const router = require("../suggest");
+
+const app = new Koa();
+app.use(router.routes());
+
+function getSuggestions(query) {
+  return request(app.callback()).get(`/api/v1/suggest?q=` + query);
+}
+
+test("return suggestions for re in the right format", () =>
+  getSuggestions("re")
+    .expect(200)
+    .expect("Content-Type", /json/)
+    .expect(res => expect(res.body).toMatchSnapshot()));
+
+test("accentuation is ignored", async () =>
+  getSuggestions("ré").expect(res =>
```

Comment:
you could also use the async await mecanism here in order to avoid having multiple `.expect`but not 100% sure.

Reply:
One expect comes from supertest and the other one is from jest, no ?
I changed this anyway according to your other comment, feel free to resolve if it looks good to you.